rules/windows/sysmon/sysmon_regedit_export_to_ads.yml

```diff
@@ -0,0 +1,25 @@
+title: Exports Registry Key To an Alternate Data Stream
+id: 0d7a9363-af70-4e7b-a3b7-1a176b7fbe84
+status: experimental
+description: Exports the target Registry key and hides it in the specified alternate data stream.
+references:
+    - https://github.com/LOLBAS-Project/LOLBAS/blob/master/yml/OSBinaries/Regedit.yml
+    - https://gist.github.com/api0cradle/cdd2d0d0ec9abb686f0e89306e277b8f
+tags:
+    - attack.defense_evasion
+    - attack.t1564.004
+author: Oddvar Moe, Sander Wiebing, oscd.community
+date: 2020/10/07
+logsource:
+    product: windows
+    service: sysmon
+detection:
+    selection:
+        EventID: 15
+        Image|endswith: '\regedit.exe'
+    condition: selection
+fieds:
```

Comment:
Hey @SanWieb, there is a typo here, without it, the rules looks ready for me.

Reply:
Thanks a lot @w0rk3r.

The test is still failing, do you have an idea why?

Reply:
Every word in the title must start with a capital letter, that is the only reason that I can thin rn

Reply:
Words like 'an' don't have to be a capital teller as far as I know. (https://capitalizemytitle.com/). Like this [4281cb20-2994-4580-aa63-c8b86d019934](https://github.com/Neo23x0/sigma/blob/master/rules/windows/process_creation/win_attrib_hiding_files.yml) rule for example.

But thank you for thinking along, maybe someone else knows what is going wrong here.

Reply:
Makes sense, looking at the logs, we can see that the test is breaking at the tags:

2020-10-15T13:42:57.0261255Z   File "/home/runner/work/sigma/sigma/tools/sigma/sigma2attack.py", line 32, in main
2020-10-15T13:42:57.0262161Z     for tag in tags:
2020-10-15T13:42:57.0263816Z TypeError: 'NoneType' object is not iterable

But there is nothing wrong with your rule, I'm going to see if I can figure out what's going on...i think you can mark this review as resolved.